<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Number of threads: </string>
           </property>
           <property name="buddy">
            <cstring>mJobs</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mJobs">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
           <property name="inputMask">
            <string notr="true">009; </string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="maxLength">
            <number>3</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Ideal count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="mLblIdealThreads">
           <property name="text">
            <string notr="true">TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="mForce">
         <property name="text">
          <string>Force checking all #ifdef configurations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mShowFullPath">
         <property name="text">
          <string>Show full path of files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mShowNoErrorsMessage">
         <property name="text">
          <string>Show &quot;No errors found&quot; message when no errors found</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mShowErrorId">
         <property name="text">
          <string>Display error Id in column &quot;Id&quot;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mInlineSuppressions">
         <property name="text">
          <string>Enable inline suppressions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mEnableInconclusive">
         <property name="text">
          <string>Check for inconclusive errors also</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mShowStatistics">
         <property name="text">
          <string>Show statistics on check completion</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="mShowDebugWarnings">
         <property name="text">
          <string>Show internal warnings in log</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Paths</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Include paths:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QListWidget" name="mListIncludePaths">
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QPushButton" name="mBtnAddIncludePath">
             <property name="text">
              <string>Add...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mBtnEditIncludePath">
             <property name="text">
              <string>Edit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mBtnRemoveIncludePath">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Applications</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QListWidget" name="mListWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="mBtnAddApplication">
           <property name="text">
            <string>Add...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mBtnEditApplication">
           <property name="text">
            <string>Edit...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mBtnRemoveApplication">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mBtnDefaultApplication">
           <property name="text">
            <string>Set as default</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Reports</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="mSaveAllErrors">
         <property name="text">
          <string>Save all errors when creating report</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mSaveFullPath">
         <property name="text">
          <string>Save full path to files in reports</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Language</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QListWidget" name="mListLanguages">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="mButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>mJobs</tabstop>
  <tabstop>mForce</tabstop>
  <tabstop>mShowFullPath</tabstop>
  <tabstop>mShowNoErrorsMessage</tabstop>
  <tabstop>mInlineSuppressions</tabstop>
  <tabstop>mListIncludePaths</tabstop>
  <tabstop>mBtnAddIncludePath</tabstop>
  <tabstop>mBtnEditIncludePath</tabstop>
  <tabstop>mBtnRemoveIncludePath</tabstop>
  <tabstop>mListWidget</tabstop>
  <tabstop>mBtnAddApplication</tabstop>
  <tabstop>mBtnEditApplication</tabstop>
  <tabstop>mBtnRemoveApplication</tabstop>
  <tabstop>mBtnDefaultApplication</tabstop>
  <tabstop>mSaveAllErrors</tabstop>
  <tabstop>mSaveFullPath</tabstop>
  <tabstop>mListLanguages</tabstop>
  <tabstop>mEnableInconclusive</tabstop>
  <tabstop>mShowStatistics</tabstop>
  <tabstop>mShowDebugWarnings</tabstop>
  <tabstop>mButtons</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mButtons</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mButtons</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
